receiver/splunkenterprisereceiver/factory.go

```diff
@@ -28,14 +28,17 @@ func createDefaultConfig() component.Config {
 	httpCfg.Headers = map[string]configopaque.String{
 		"Content-Type": "application/x-www-form-urlencoded",
 	}
+	httpCfg.Timeout = defaultMaxSearchWaitTime
```

Comment:
Was this just not being set before at all? Or was it pulling the default from the httpclient or something?